packages/core/src/use-params.ts

```diff
@@ -97,6 +97,11 @@ export function useParam(
 ): undefined | number | string | Array<string | undefined> {
   const params = useParams()
   const rawValue = params[key]
+  const exists = params.hasOwnProperty(key)
```

Comment:
possibly

```suggestion
  const exists = params[key] !== undefined
```
or

```suggestion
  const exists = params[key] != null
```

Reply:
You can also try the following since it will handle case for rawValue being undefined and null.

``
const exists = !!rawValue
``

I also think it's cleaner to set `exists` before accessing the rawValue. The way it is now, we don't need to call hasOwnProperty like you suggested above if we know rawValue is falsey.

Reply:
No, falsey is not correct - it will break empty strings.

https://github.com/blitz-js/blitz/blob/9819543f47cc03c52eaceb829bbdeff441aeb9e7/packages/core/test/use-params.test.ts#L116-L117

It think it has to be either hasOwnProperty (above), strict undefined check (above) or non-strict null-check (above), just pick one of those :) 

> You can also try the following since it will handle case for rawValue being undefined and null.

`!= null` does just that - but the `null`-check is redundant which is why I also proposed `!== undefined`

Reply:
Thanks for catching that